docs/blog/articles/2024-12-13-KServe-0.14-release.md

```diff
@@ -0,0 +1,149 @@
+
+The Model Cache was proposed as another alternative to enhance KServe usability with big models, released in KServe v0.14 as an **alpha** feature. It relies on a PV for storing models and it provides control about which models to store in the cache. The feature was designed to mainly to use node Filesystem as storage. Read the [design document for the details](https://docs.google.com/document/d/1nao8Ws3tonO2zNAzdmXTYa0hECZNoP2SV_z9Zg0PzLA/edit).
+
+The model cache is currently disabled by default. To enable, you need to modify the `localmodel.enabled` field on the `inferenceservice-config` ConfigMap.
+
+You start by creating a node group as follows:
+
+```yaml
+apiVersion: serving.kserve.io/v1alpha1
+kind: LocalModelNodeGroup 
+metadata:
+  name: nodegroup1
+spec:
+  persistentVolumeSpec:
+    accessModes:
+    - ReadWriteOnce
+    volumeMode: Filesystem
+    capacity:
+      storage: 2Gi
+    hostPath:
+      path: /models
+      type: ""
+    persistentVolumeReclaimPolicy: Delete
+    storageClassName: standard
+  persistentVolumeClaimSpec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 2Gi
+    storageClassName: standard
+    volumeMode: Filesystem
+    volumeName: kserve
+
+```
+
+Then, you can specify to store an cache a model with the following resource:
+
+```yaml
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterLocalModel
+metadata:
+  name: iris
+spec:
+  modelSize: 1Gi
+  nodeGroup: nodegroup1
+  sourceModelUri: gs://kfserving-examples/models/sklearn/1.0/model
+```
+
+<!--
+Related tickets:
+* Cluster local model controller: [#3860](https://github.com/kserve/kserve/pull/3860)
+* Cluster Local Model CR [#3839](https://github.com/kserve/kserve/pull/3839)
+* Add NodeDownloadPending status to ClusterLocalModel [#3955](https://github.com/kserve/kserve/pull/3955)
+-->
+
+## Support for Hugging Face hub in storage initializer
+
+The KServe storage initializer has been enhanced to support downloading models directly from Hugging Face. For this, the new schema `hf://` is now supported in the `storageUri` field of InferenceServices. The following YAML partial shows this:
+
+```yaml
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: huggingface-llama3
+spec:
+  predictor:
+    model:
+      storageUri: hf://meta-llama/meta-llama-3-8b-instruct
+```
+
+Both public and private Hugging Face repositories are supported. The credentials can be provided by the usual mechanism of binding Secrets to ServiceAccounts, or by binding the credentials Secret as environment variables in the InferenceService.
+
+Read the [documentation](../../../modelserving/storage/huggingface/hf/) for more details.
+
+<!--
+Related tickets:
+* Implement Huggingface model download in storage initializer [#3584](https://github.com/kserve/kserve/pull/3584)
+-->
+
+## Other Changes
+
+This release also includes several enhancements and changes:
+
+### What's New?
```

Comment:
@sivanantha321 @andyi2it also good to the add the binary extension support and response header support?
https://github.com/kserve/website/pull/419

Reply:
I somehow thought that the binary extension was an enhancement of the Inference Client. So, to better understand... should I add it under the inference client heading? or it is good here as a bullet under `What's New?` ?

Reply:
binary extension is not part of the inference client effort, it is implementing the binary extension as part of the open inference protocol along with FP16 support.

Reply:
Maybe you can link the documentation as well. https://kserve.github.io/website/latest/modelserving/data_plane/binary_tensor_data_extension/